api/src/test/java/org/pmiops/workbench/actionaudit/ActionAuditQueryServiceTest.java

```diff
@@ -56,8 +57,8 @@
   private static final long AGENT_ID = 202L;
   private static final String ACTION_ID_1 = "abfcb9ed-fa65-4e98-acb2-08b0d8b30000";
   private static final String USERNAME = "jay@unit-test-aou.org";
-  private static final DateTime EVENT_DATETIME = DateTime.parse("2010-06-30T01:20+02:00");
-  public static final long EVENT_TIME_SECONDS = EVENT_DATETIME.getMillis() / 1000;
+  private static final Instant EVENT_INSTANT = Instant.parse("2010-06-30T01:20:00.00Z");
```

Comment:
Should this be `.02Z` like the others?

Reply:
No, the instant doesn't need to know what my after/before times are.